src/index.spec.ts

```diff
@@ -1,7 +1,20 @@
 import expect from "expect";
+import { getInitialBoard } from './Board'
+import { buildGraphicalBoard } from './VisualBoard'
 
 describe("initial test", () => {
   it("should pass", () => {
     expect(true).toBe(true);
   });
+
```

Comment:
You should put these tests in another file. Try to have a test file structure that mimics the code file structure